docs/rep/node-agent/README.md

```diff
@@ -0,0 +1,99 @@
+# RIK - Node Agent Architecture (Riklet)
+
+This document provides a technical reflection about our component, the **node agent**.
+
+Our component will be installed on each node of a cluster and it will be composed of 4 sub-components :
+
+- **Node Manager**
+- **Node Runtime Memory**
+- **Node Proxy**
+- **Node Monitor**
+
+We will describe these components in more details throughout this document. Here is a diagram to illustrate how these components works together :
+
+![High-level view of Rikelet](assets/node-schema.png)
+
+## 1) Node Manager
+
+The Node Manager is the main sub-component of the Riklet. It should be able to :
+
+- Interprete Scheduler instructions
+- Run / Stop an application
+- List pods in the node
+
+## 2) Node Runtime Memory
+
+![](assets/nrm.png)
+
+The Node Runtime Memory is a simple runtime memory which allow us to manage all the running pods on the node. This component allow the Node Manager to get Pods instance fastly to execute actions on them. It saves the state of each pods.
+
```

Comment:
Is the sole role of that component to store and load a pod state? Is any other component but the node manager going to interact with it? 
I suggest to either rename that component to `Pod state manager` or simply integrate it into the node manager.

Reply:
You are right, we'll simply integrate it into the node manager.